en/style/colors.md

```diff
@@ -1,18 +1,20 @@
 ---
 title: Colors - Design System Styling
-_description: The Colors library page is the go to place for setting up a theme for the Indigo.Design System.
+_description: The color styles let you set up palettes and special colors as the basis of a theme for the Indigo.Design System.
 _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
 ---
 
 # Colors
 
-On the `🎨 Colors` page of the **Indigo.Design System** you will find a collection of `primary`, `secondary`, `success`, `warn`, `error`, `info`, `grays`, and `series` colors to let you brand the library all at once. The way Colors are set up is identical to the [Ignite UI for Angular Themes](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes.html).
+There are five palettes with 10 color variations each for the `primary`, `secondary`, `grays`, and two chart `sereis` colors, as well as special colors such as `success`, `warn`, `error`, `info`, and `surface` in the the **Indigo.Design System**. This lets you change the complete look of the library and all projects that use it with just a few clicks. The way Colors are set up is identical to the [Ignite UI for Angular Themes](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes.html).
 
 <img class="responsive-img" src="../images/colors_palette.png" />
 
-The **Indigo.Design System** version 3 introduces the concept of series palettes with two collections of ten colors each optimized for various data visualization scenarios. Both palettes ensure good contrast between adjacent series colors so that the palette can be used e.g. to colorize ranges of a gauge. One of the palettes is further optimized to be 100% inclusive and facilitate easy comprehension by people with color vision defficiencies. 
+In Sketch, these can be found on a separate page named `🎨 Colors` and are available as `Layer Styles` that can be applied to any shape element. In Adobe XD they are placed on an Artboard with the same name in the left column of the library file and exist as `Colors` that are used across components and patterns.
 
-## Palette Generation
+The **Indigo.Design System** features special color palettes optimized for various data visualization scenarios. Both ensure good contrast between adjacent series colors so that the palette can be used e.g. to colorize ranges of a gauge. One of the palettes is further optimized to be 100% inclusive and facilitate easy comprehension by people with color vision defficiencies. 
```

Comment:
```suggestion
The **Indigo.Design System** features special color palettes optimized for various data visualization scenarios. Both ensure good contrast between adjacent series colors so that the palette can be used e.g. to colorize ranges of a gauge. One of the palettes is further optimized to be 100% inclusive and facilitate easy comprehension by people with color vision deficiencies. 
```